diff -ur a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2026-08-13 11:27:31.892449716 -0400
+++ b/CMakeLists.txt	2026-08-13 11:28:29.732829798 -0400
@@ -324,14 +324,9 @@
 install (FILES "${OSL_PROJECT_CONFIG}" "${OSL_VERSION_CONFIG}"
         DESTINATION "${OSL_CONFIG_INSTALL_DIR}")
 
-install (FILES src/cmake/llvm_macros.cmake DESTINATION cmake)
-
 set (PERMISSION_FLAGS OWNER_EXECUTE OWNER_READ OWNER_WRITE
                       GROUP_EXECUTE GROUP_READ
                       WORLD_EXECUTE WORLD_READ)
-install (FILES src/build-scripts/serialize-bc.py
-         DESTINATION build-scripts
-         PERMISSIONS ${PERMISSION_FLAGS})
 
 # install targets files
 install (EXPORT OSL_EXPORTED_TARGETS
